proposals/dao/buyback_newpool.ts

```diff
@@ -0,0 +1,149 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { TransactionResponse } from '@ethersproject/providers';
+import { expectApprox } from '@test/helpers';
+import { getImpersonatedSigner } from '@test/helpers';
+
+chai.use(CBN(ethers.BigNumber));
+const toBN = ethers.BigNumber.from;
+const e18 = ethers.constants.WeiPerEther;
+
+// LBP swapper
+const LBP_FREQUENCY = '604800'; // weekly
+const MIN_LBP_SIZE = ethers.constants.WeiPerEther.mul(100_000); // 100k FEI
+
+/*
+
+TRIBE Buybacks
+
+DEPLOY ACTIONS:
+
+1. Deploy TRIBE LBP Swapper
+2. Create TRIBE LBP pool
+3. Init TRIBE LBP Swapper
+
+DAO ACTIONS:
+1. Set PCVEquityMinter target to new buyback swapper
+2. Exit buyback liquidity from old swapper to new swapper
+3. Mint 4m FEI for missed buybacks and this week's buybacks
+4. Unpause the PCV Equity Minter
+5. Re-start the buybacks
+*/
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  if (!addresses.core) {
+    throw new Error('An environment variable contract address is not set');
+  }
+
+  // 1.
+  const BalancerLBPSwapperFactory = await ethers.getContractFactory('BalancerLBPSwapper');
+  const noFeeFeiTribeLBPSwapper = await BalancerLBPSwapperFactory.deploy(
+    addresses.core,
+    {
+      _oracle: addresses.tribeUsdCompositeOracle,
+      _backupOracle: ethers.constants.AddressZero,
+      _invertOraclePrice: true,
+      _decimalsNormalizer: 0
+    },
+    LBP_FREQUENCY,
+    addresses.fei,
+    addresses.tribe,
+    addresses.core, // send TRIBE back to treasury
+    MIN_LBP_SIZE
+  );
+
+  await noFeeFeiTribeLBPSwapper.deployTransaction.wait();
+
+  logging && console.log('FEI->TRIBE LBP Swapper: ', noFeeFeiTribeLBPSwapper.address);
+
+  // 2.
+  const lbpFactory = await ethers.getContractAt(
+    'ILiquidityBootstrappingPoolFactory',
+    addresses.balancerLBPoolFactoryNoFee
+  );
+
+  const tx: TransactionResponse = await lbpFactory.create(
+    'FEI->TRIBE Auction Pool',
+    'apFEI-TRIBE',
+    [addresses.fei, addresses.tribe],
+    [ethers.constants.WeiPerEther.mul(90).div(100), ethers.constants.WeiPerEther.mul(10).div(100)],
+    ethers.constants.WeiPerEther.mul(30).div(10_000),
```

Comment:
what is this param? Can you add comments?

Reply:
this is the fee, the pool has 0.3% swap fees